packages/keyring-controller/src/KeyringController.ts

```diff
@@ -441,6 +441,20 @@ export class KeyringController extends BaseControllerV2<
     return this.#keyring.getKeyringForAccount(account);
   }
 
+  /**
+   * Returns the encryption public key of
+   * the specified `address` if one exists.
+   *
+   * @deprecated Use of this method is discouraged as actions executed directly on
+   * keyrings are not being reflected in the KeyringController state and not
+   * persisted in the vault.
```

Comment:
We don't need to deprecate this one, as the client calling this method will not have direct access to the underlying EthKeyringController or Keyring.

Moreover, `this.#keyring.getEncryptionPublicKey(account)` does not apply any change to the controller state